src/backend/expungeservice/expunger/analyze.py

```diff
@@ -91,6 +96,14 @@ def __init__(
         self.date = date
         self.disposition = disposition
 
+    @property
+    def type_elig_result(self):
+        return self._result
+
+    @type_elig_result.setter
+    def type_elig_result(self, result):
+        self._result = result
```

Comment:
This should probably be instantiated within the constructor, even if it's just being instantiated to None.